CHANGELOG.md

```diff
@@ -10,6 +10,11 @@ Every entry has a category for which we use the following visual abbreviations:
 
 ## [Unreleased]
 
+- 🐞 Previously, timestamps were always printed in millisecond resolution.
+  Hence, it was possible that a query produced no result after copying and
+  pasting a timestamp from a previous query result. Timestamps are now printed
+  in the resolution necessary, down to nanoseconds.
```

Comment:
Here's an alternate wording suggestion:

```
Previously, timestamps were always printed in millisecond resolution.
Timestamps were always printed in millisecond resolution, which lead to loss of
precision when the internal representation had a higher resoultion. Timestamps
are now rendered up to nanosecond resolution - the maximum resolution
supported.
```

Reply:
The first sentence gets repeated in your suggestion. I suppose the "Previously, ..." sentence was an oversight?

Reply:
Whoops, right. Here's the proper text:
```
Timestamps were always printed in millisecond resolution, which lead to loss of
precision when the internal representation had a higher resolution. Timestamps
are now rendered up to nanosecond resolution - the maximum resolution
supported.
```

Also, some more explanation for the rewording: what I wanted to change here ist the focus on the bug (= loss of precision). The use case with copy-and-paste confused me initially. I'm sure there's also many other cases where the precision matters.